cedar-policy-cli/src/main.rs

```diff
@@ -53,5 +53,6 @@ fn main() -> CedarExitCode {
         Commands::TranslateSchema(args) => translate_schema(&args),
         Commands::New(args) => new(&args),
         Commands::PartiallyAuthorize(args) => partial_authorize(&args),
+        Commands::LanguageVersion => language_version(),
```

Comment:
I think we should have a single `version` command that prints both the language & SDK versions.

Reply:
(But aside from that I'm happy with this PR)

Reply:
Yeah. Users can get the SDK version just using `--version`.